Composer/packages/client/src/pages/setting/index.tsx

```diff
@@ -104,7 +104,7 @@ const SettingPage: React.FC<RouteComponentProps<{ '*': string }>> = () => {
     };
     const res = await OpenConfirmModal(title, null, settings);
     if (res) {
-      actions.deleteBotProject(projectId);
+      await actions.deleteBotProject(projectId);
       navigateTo('home');
```

Comment:
When navigating to home page, we will fetch the recent bot lists. But the recent bot list is probably not updated because we did not wait on the http request. Adding await will solve the problem.